### ISSUE TITLE
`npm install -g meteor` has overly strict install requirements

### ISSUE THREAD

**Issue by Julusian** [meteor/meteor#13008]
When running `npm install -g meteor` to install the meteor command line tools, I have to be using node 14 or older.
Once this has been run, you can switch to newer versions of nodejs (and can even uninstall the 14 version), and meteor doesn't care.  
This just affects the installer, real meteor will use the version of nodejs that it has bundled.

This limitation is causing onboarding guides to be unnecessarily complex. It essentially requires developers to do the following:
* `nvm install 14`
* `npm install -g meteor`
* `nvm install 18`
* The rest of the setup stuff.
Where they have to install node 14 just to run one command, and then switch to the version of node we use for the rest of our non-meteor applications. 

Surely the installer shouldn't care what the system version will be? as long as it is capable of running on the current version without issue

As an additional thought, it seems weird to me to install a npm package to let it run a script and then never use it again. It would be nicer for this be reworked to be a `npx` based command, but that isn't as important to change.

**Comment by StorytellerCZ** [comment]
Strange, this is the current setup:
```
"engines": {
    "node": "<=14.x",
    "npm": "<=6.x"
  }
```

Maybe changing it to:
```
"engines": {
    "node": "<=14",
    "npm": "<=6"
  }
```
might fix the issue?

**Comment by Julusian** [comment]
No, I want to be able to run `npm install -g meteor` while using node 18 or 20. Ideally there wouldn't be an engines field at all, as this field only affects this installer, and that shouldn't need to run with the same version of node as meteor itself

**Comment by Julusian** [comment]
I guess this has gone away since 3.0 was released, and my usage has updated to 3.x
